datafusion/sqllogictest/test_files/group_by.slt

```diff
@@ -3214,7 +3214,7 @@ JOIN sales_global AS e
   ON s.currency = e.currency AND
   s.ts >= e.ts
 GROUP BY s.sn, s.zip_code, s.country, s.ts, s.currency
-ORDER BY s.sn
+ORDER BY s.sn, s.zip_code
```

Comment:
The expected result is
```
0 GRC 0 2022-01-01T06:00:00 EUR 30
1 FRA 1 2022-01-01T08:00:00 EUR 50
1 TUR 2 2022-01-01T11:30:00 TRY 75
1 FRA 3 2022-01-02T12:00:00 EUR 200
0 GRC 4 2022-01-03T10:00:00 EUR 80
1 TUR 4 2022-01-03T10:00:00 TRY 100
```
but my test has
```
0 GRC 0 2022-01-01T06:00:00 EUR 30
1 FRA 1 2022-01-01T08:00:00 EUR 50
1 TUR 2 2022-01-01T11:30:00 TRY 75
1 FRA 3 2022-01-02T12:00:00 EUR 200
1 TUR 4 2022-01-03T10:00:00 TRY 100
0 GRC 4 2022-01-03T10:00:00 EUR 80
```
I think the above is also legal. I added one more column in `ORDER BY` to make the returned order deterministic.